tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts

```diff
@@ -0,0 +1,102 @@
+import { Logger } from "winston";
+import { DockerCliConfig } from "../schema/dockerCliConfig";
+import { initializeLogger } from "@azure-tools/sdk-generation-lib";
+import path from "path";
+import fs from "fs";
+import { sdkToRepoMap } from "./generateCodesInLocal";
+
+export class DockerContext {
+    mode: 'generateCodesInLocal' | 'growUp' | 'generateCodesInPipeline';
+    readmeMdPath?: string;
+    tag?: string;
+    sdk: string[];
+    specRepo?: string;
+    workDir?: string;
```

Comment:
is it better to use workingDir?

Reply:
It's related to the command customer need to input. I hope it should as short as possible.